app/views/helpers/index/normal/entry_header.phtml

```diff
@@ -76,14 +77,25 @@
 	endif; ?>
 	<?php if ($topline_date) { ?><span class="item-element date"><time datetime="<?= $this->entry->machineReadableDate() ?>"><?= $this->entry->date() ?></time>&nbsp;</span><?php } ?>
 	</li>
+	<?php if ($topline_myLabels) { ?>
+		<li class="item labels">
+			<div class="item-element dropdown dynamictags">
+				<div id="dropdown-labels2-<?= $this->entry->id() ?>" class="dropdown-target"></div>
```

Comment:
Is there maybe a similar ID-uniqueness issue there than https://github.com/FreshRSS/FreshRSS/pull/6990 ?

Reply:
No. In `entry_header.phtml`: `dropdown-labels2`. In `entry_bottom.phtml`: `dropdown-labels`.